src/System.Private.ServiceModel/tests/Scenarios/Binding/Http/BasicHttpBindingTests.cs

```diff
@@ -16,32 +16,32 @@ public static class Binding_Http_BasicHttpBindingTests
     [OuterLoop]
     public static void DefaultSettings_Echo_RoundTrips_String()
     {
-        string variationDetails = "Client:: BasicHttpBinding/DefaultValues\nServer:: BasicHttpBinding/DefaultValues";
+        // *** ASSEMBLE *** \\
+        ChannelFactory<IWcfService> factory = null;
```

Comment:
Nit: the terms the WebAPI uses here are "Arrange", "Act" and "Assert".  We are free to choose our own of course but I recommend we follow some convention.  Personally I think I like "Setup", "Act", "Validate" and "Cleanup".  I have no strong opinion for the "Force Cleanup".